lib/nerve/reporter/zookeeper.rb

```diff
@@ -124,6 +125,21 @@ def host_list_to_cluster(list)
       last_non_number ? first_token[0..last_non_number] : first_host
     end
 
+    def get_key(service)
+      if service.has_key?('use_path_encoding') && service['use_path_encoding'] == true
+        obj = {
+          'host' => service['host'],
+          'port' => service['port']
+        }
+        if service.has_key?('labels')
+          obj['labels'] = service['labels']
```

Comment:
there might be other possible values, like `haproxy_server_options` inside zk node. Instead of hard code options, could we just copy all key into the base 64?

Reply:
![image](https://user-images.githubusercontent.com/19193590/61251285-d36c9480-a70e-11e9-9198-bab3f3a1c2d5.png)

Reply:
otherwise, any additional fields won't be included in encoded result